std/stdio.d

```diff
@@ -17,13 +17,12 @@ Authors:   $(WEB digitalmars.com, Walter Bright),
 module std.stdio;
 
 public import core.stdc.stdio;
-static import core.stdc.stdio;
-import std.typecons : Flag;
```

Comment:
I laid out the reason in my second comment above.  Issue 314 is that selective imports at module scope are currently always public, ie those symbols are leaked into any module that imports this module.  Since this selective import was added between major releases, I thought it best to reverse it, until issue 314 is fixed, so that it won't leak in the next release.